elrond_dex_farm/src/lib.rs

```diff
@@ -220,6 +221,12 @@ pub trait Farm {
             entering_epoch: self.blockchain().get_block_epoch(),
         };
 
+        // Do the actual permanent lock of first minimul liquidity
+        // only after the token attributes are crafted for the user.
+        if is_first_provider {
+            liquidity -= BigUint::from(self.liquidity_pool().minimul_liquidity_farm_amount());
```

Comment:
I think you have a typo here: `minimul_liquidity_farm_amount`
Shouldn't it be `minimum_liquidity_farm_amount` ?